### ISSUE TITLE
Accessibility of the Assets section

### ISSUE THREAD

**Issue by ajanec01** [craftcms/cms#7377]
### Overview of accessibility issues in the Assets section

- The display buttons (table/ thumbnails) function as radio buttons so it would be better to make them a group of radio buttons with a legend (at the moment they don’t convey the selected state)
- Thumbnails can be navigated using arrow keys, they resemble radio buttons but do not have the necessary functionality- confusing to sr users
- When using 400% zoom in, only one row this visible; not a failure of reflow but the elements above the table could be stacked a bit better; it's a bit more problematic for devices with 320px wide viewport. 
![reflow](https://user-images.githubusercontent.com/47136246/104526945-80bdb280-55fb-11eb-87b0-71def40bbcbc.png)
- Preview dialog doesn’t display any content
- I’m not sure what the new heading section does, I typed in a name for a new heading and hit save but nothing has happened (I was expecting a new column in the table)
- Copy url and copy reference tag dialogs do not have an accessible name and accessible role 
![copy-url-focus](https://user-images.githubusercontent.com/47136246/104254285-6904f380-546e-11eb-80b6-8435cae64ddf.png)
- Also sidebar issues already covered in #7308 

### Additional info

- Craft version: 3.5.17.1

**Comment by gcamacho079** [comment]
I'll be closing this issue as stale as a couple of the mentioned issues have been resolved as of 4.3, or are a part of more intensive component updates on our roadmap. Resolved issues include:

- The display buttons now implement a pattern for exclusive button components, and convey their state via `aria-pressed` attributes
- Modals opened via the Actions button have accessible roles and names

I was unable to reproduce the Preview dialog not displaying content. 🤔 

Regarding the **New heading** action, it creates a new source group in the navigational sidebar. However, the new heading will only show if a source is dragged underneath it while you're still in the **Customize sources** modal.
